admin/src/components/ContributionMessages/slots/ContributionMessagesListItem.vue

```diff
@@ -1,21 +1,34 @@
 <template>
   <div class="contribution-messages-list-item">
     <div v-if="message.isModerator" class="text-right is-moderator">
-      <b-avatar square :text="initialLetters" variant="warning"></b-avatar>
+      <b-avatar square variant="warning"></b-avatar>
       <span class="ml-2 mr-2">{{ message.userFirstName }} {{ message.userLastName }}</span>
       <span class="ml-2">{{ $d(new Date(message.createdAt), 'short') }}</span>
       <small class="ml-4 text-success">{{ $t('moderator') }}</small>
-      <div class="mt-2">{{ message.message }}</div>
+      <div class="mt-2">
+        <span v-for="({ type, text }, index) in linkifiedMessage" :key="index">
+          <b-link v-if="type === 'link'" :to="text">{{ text }}</b-link>
+          <span v-else>{{ text }}</span>
+        </span>
+      </div>
     </div>
     <div v-else class="text-left is-not-moderator">
-      <b-avatar :text="initialLetters" variant="info"></b-avatar>
+      <b-avatar variant="info"></b-avatar>
       <span class="ml-2 mr-2">{{ message.userFirstName }} {{ message.userLastName }}</span>
       <span class="ml-2">{{ $d(new Date(message.createdAt), 'short') }}</span>
-      <div class="mt-2">{{ message.message }}</div>
+      <div class="mt-2">
+        <span v-for="({ type, text }, index) in linkifiedMessage" :key="index">
+          <b-link v-if="type === 'link'" :to="text">{{ text }}</b-link>
+          <span v-else>{{ text }}</span>
+        </span>
+      </div>
     </div>
   </div>
 </template>
 <script>
+const PATTERN =
```

Comment:
Please describe this better so one can search for this:

```suggestion
const LINK_REGEX_PATTERN =
```

```suggestion
const LINK_REGEX =
```